lib/gc/ExecutionEngine/OpenCLRuntime/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+find_package(OpenCL REQUIRED)
+
+if(NOT OpenCL_FOUND)
+    message(FATAL_ERROR "OpenCL not found.")
+endif()
+
+add_mlir_library(opencl-runtime
```

```suggestion
add_mlir_library(GcOpenclRuntime
```
Let's have consistent targets naming. It should help to avoid name clashes when integrating with other projects.

---

To be consistent with the upstream it should actually be `mlir_opencl_runtime`.

---

ok. changed to `mlir_opencl_runtime`.